Pages/documentation.css

```diff
@@ -147,6 +147,14 @@ header .navbar a:hover{
     width: fit-content;
 }
 
+.component-container-nav{
+    margin-top: 2rem;
+    margin-bottom: 2rem;
```

Comment:
Here instead of giving margin top and bottom separately you can use the shorthand margin : 2rem 0. Very minute change everything else looks great

Reply:
Thanks ! will look into it